Assets/Controllers/GameManager.cs

```diff
@@ -24,18 +26,32 @@ public void StartPressed()
         hexMap = FindObjectOfType<HexMap>();
         pawnKeeper = FindObjectOfType<PawnKeeper>();
         players = new List<Player>();
-        string newPlayerName = "Syd";
-        Player newPlayer = new Player(newPlayerName);
-        players.Add(newPlayer);
-        activePlayer = newPlayer;
+        for (int i = 0; i < startingNumberOfPlayers; i++)
```

Comment:
The hardcoded Syd is a placeholder for when I implement the UI to name players.   The names are actually Syd_0 through Syd_whateverthemaxnumberofplayersare-1.

Agreed on it making a fine method, though, so that's been changed.